Basis/Packages/com.basis.framework/UI/UI Panels/BasisHamburgerMenu.cs

```
@@ -63,7 +65,8 @@ public override void DestroyEvent()
         }
         private void OnBootModeChanged(string obj)
         {
-            FullBodyParent.SetActive(!BasisDeviceManagement.IsUserInDesktop());
+            if(FullBodyParent != null)
```

Hiding the error is no good.